proposals/collection-expression-arguments.md

```diff
@@ -331,14 +337,16 @@ If the target type is an *interface type*, then:
 
   |Interfaces|Candidate signatures|
   |:---:|:---:|
-  |`IEnumerable<E>`<br>`IReadOnlyCollection<E>`<br>`IReadOnlyList<E>`|*None*|
-  |`ICollection<E>`<br>`IList<E>`|`(int capacity)`|
-  |`IReadOnlyDictionary<K, V>`|`(IEqualityComparer<K>? comparer)`|
-  |`IDictionary<K, V>`|`(int capacity)`<br>`(IEqualityComparer<K>? comparer)`<br>`(int capacity, IEqualityComparer<K>? comparer)`|
+  |`IEnumerable<E>`<br>`IReadOnlyCollection<E>`<br>`IReadOnlyList<E>` `()` (no parameters)|
```

Comment:
Formatting looks wrong here, the `(no parameters)` is in the same cell as the `IEnumerable`, etc.

Reply:
```suggestion
  |`IEnumerable<E>`<br>`IReadOnlyCollection<E>`<br>`IReadOnlyList<E>`|`()` (no parameters)|
```